emrun.py

```diff
@@ -596,6 +600,9 @@ def pid_existed(pid):
     self.send_header('Connection', 'close')
     self.send_header('Expires', '-1')
     self.send_header('Access-Control-Allow-Origin', '*')
+    self.send_header('Cross-Origin-Opener-Policy', 'same-origin')
+    self.send_header('Cross-Origin-Embedder-Policy', 'require-corp')
+    self.send_header('Cross-Origin-Resource-Policy', 'cross-origin')
```

Comment:
Any particular reason you are setting this last header? If your application is same-origin setting this shouldn't be needed and would in fact be somewhat unsafe.

Reply:
should it be set to `same-site` instead, or deleted? If it is deleted, we should change COEP too, right? Sorry for not being thorough my only source is [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes)

Reply:
COEP is the policy you impose upon yourself, CORP is who you are willing to share the current resource with. CORP is not required to enable `postMessage()` of shared memory, but is required on any cross-origin resources inside a COEP environment.

So whether to set CORP depends a bit on the particulars of your deployment. I guess I should expand that wiki page a bit with more information.

Reply:
Okay, so since we are allowing all origins:
```
     self.send_header('Access-Control-Allow-Origin', '*') 
```
and keeping in mind this is not for production, should I set this instead?
```
    self.send_header('Cross-Origin-Opener-Policy', 'same-origin')
    self.send_header('Cross-Origin-Embedder-Policy', 'none')
```
and maybe add a note on the fact that if we want to allow cross origin, we would need to do it via require-corp ?

Reply:
found [this](https://mikewest.github.io/corpp/), it says that for now any value other than require-corp would get ignored.

Reply:
@annevk small ping for the review, is it okay as is? or should I do what I suggested? or something else?

Reply:
I guess it's okay, if there are no cookies involved. If there are cookies involved CORS with a wildcard would still not allow sharing their secrets, but Spectre and these other headers would.

Reply:
@annevk is there something you'd like me to add/amend ?

Reply:
No, I'm fine with this if you understand the implications of CORP. I've updated https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes to be more explicit about them. Hope that helps.